packages/clerk-js/src/core/resources/Base.ts

```diff
@@ -32,10 +32,18 @@ export abstract class BaseResource {
       clerkMissingFapiClientInResources();
     }
 
-    const { payload, status, statusText } = await runWithExponentialBackOff(
-      () => BaseResource.fapiClient.request<J>(requestInit),
-      { maxRetries: 4, firstDelay: 800 },
-    );
+    let fapiResponse: FapiResponse<J>;
+
+    // retry only on GET requests for safety
+    if (requestInit.method === 'GET') {
+      fapiResponse = await runWithExponentialBackOff(() => BaseResource.fapiClient.request<J>(requestInit), {
+        maxRetries: 4,
+        firstDelay: 800,
```

Comment:
This is too much. I'd set it to 500ms for now. What's the delay between exponential retries?

Reply:
Ok, I'll change it to 500ms, and then it will be 0, 500, 1000, 2000.